README.md

```diff
@@ -675,6 +675,22 @@ $ lerna exec --scope my-component -- ls -la
 $ lerna run --scope toolbar-* test
 ```
 
+#### --since [ref]
+
+When executing a script or command, only run the script or command on packages that have been updated since the specified `ref`. If `ref` is not specified, it defaults to the latest tag.
+
+```sh
+$ lerna exec --since -- ls -la
+```
+
+```
+$ lerna run test --since master
+```
+
+```
+$ lerna ls --since some-branch
```

Comment:
I noticed while doing my updates that `--since` when used with `lerna ls` seems to make `lerna updated` redundant (except that one shows versions and one doesn't).